pages/api/student_email_join.js

```diff
@@ -2,14 +2,36 @@ import prisma from '../../prisma/prisma';
 
 export default async function handle(req, res) {
   const data = JSON.parse(req.body);
-  const createdStudentEmail = await prisma.classroom.update({
-    where: {
-      classroomId: data['classId'][0]
-    },
-    data: {
-      fccUserIds: { push: data['email'] }
+  const userInfoReq = await fetch(
+    `http://localhost:3001/get-student-profile?email=${data['email']}`
```

Comment:
Me and Guillermo changed this to `localhost:3002/getStudentProfile` my bad, forgot that we shouldn't have dashes in urls and we also have 2 json servers running so we need the port 3002.  Merge from main after you change this and double check everything still works.

Reply:
Changed the url